packages/dashboards/package.json

```diff
@@ -148,6 +146,12 @@
         "serve-examples": "ng s examples",
         "serve-examples:prod": "npx local-web-server -p 4200 -d sdk/api-docs-ng2/examples --spa index.html",
         "start": "run-p -lr compodoc:serve build-examples:watch",
+        "symlink": "run-s symlink:*",
```

Comment:
I guess you need to add it to "assemble" or are you going to change the build config?